docker/docker-compose.yml

```diff
@@ -29,7 +29,8 @@ services:
       - 8080:8080
       - 5005:5005
     volumes:
-      - ../target/DeveloperApi.war:/usr/local/tomcat/webapps/DeveloperApi.war
+      - ../DeveloperApi/:/usr/local/tomcat/webapps/
```

Comment:
Maybe it is better to create a separate dir in the target folder and use it?

Reply:
fixed

Reply:
Good solution! I like it!